plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveWriterFactory.java

```diff
@@ -34,4 +35,24 @@ public void testComputeBucketedFileName()
         assertEquals(name, "001234_0");
         assertEquals(getBucketIdFromFile(name), 1234);
     }
+
+    @Test
+    public void testSetsSchemeToFile()
+    {
+        String pathWithoutScheme = "/simple/file/path";
+        String result = setSchemeToFileIfAbsent(pathWithoutScheme);
+        assertThat(result).isEqualTo("file:////simple/file/path");
```

Comment:
Please assert that you can actually create `Path` from `result`
Please assert on:
* `new Path(result).toUri().getScheme()`
* `new Path(result).toUri().getPath()`

here and for other assertions